.travis.yml

```diff
@@ -1,18 +1,22 @@
 version: "~> 1.0"
```

Comment:
Interesting, so to test the dotnet related commands, you are installing node on a dotnet/csharp node. This works for now, but does bring up the point that we could very well get into a scenario where we need to have multiple travis build environments as the CLI works with even other project types. No need to go down that road now of course.

Reply:
For sure. I wasn't sure if it would be better to manually install the dotnet dependencies in the script itself, or use the community dotnet template. I chose the path of less resistance for now 😛